tests/lucid.test.ts

```diff
@@ -0,0 +1,116 @@
+import { describe, expect, it, vi } from 'vitest'
+import { promises as fs } from 'fs';
+import {
+  Assets, 
+  ConstrData, 
+  MintingPolicyHash,
+  NetworkEmulator,
+  NetworkParams,
+  Program, 
+  Value, 
+  textToBytes,
+  TxOutput,
+  Tx, 
+} from "@hyperionbt/helios";
+
+import {
+  Assets,
+  Data,
+  Emulator,
+  fromText,
+  generateSeedPhrase,
+  getAddressDetails,
+  Lucid,
+  SpendingValidator,
+  toUnit,
+  TxHash,
+} from "lucid-cardano"; // NPM
+
+describe('Verbose test', () => {
+
+    const main = async () => {
+
+	// Set the Helios compiler optimizer flag
+        let optimize = false;
+        const minAda = BigInt(2000000);  // minimum lovelace needed to send an NFT
+
+        try {
+	// https://github.com/spacebudz/lucid/blob/main/tests/emulator.test.ts
+	async function generateAccount(assets: Assets) {
+	  const seedPhrase = generateSeedPhrase();
+	  return {
+	    seedPhrase,
+	    address: await (await Lucid.new(undefined, "Custom"))
+	      .selectWalletFromSeed(seedPhrase).wallet.address(),
+	    assets,
+	  };
+	}
+
+	const lender = await generateAccount({ lovelace: 75000000000n });
+	const borrower = await generateAccount({ lovelace: 100000000n });
+
+	const emulator = new Emulator([lender, borrower]);
+
+	const lucid = await Lucid.new(emulator);
+
+	lucid.selectWalletFromSeed(borrower.seedPhrase);
+	
+	// https://lucid.spacebudz.io/docs/getting-started/mint-assets/
+	const recipient = await lucid.wallet.address();
+	const { paymentCredential } = lucid.utils.getAddressDetails( recipient );
+
+	const mintingPolicy = lucid.utils.nativeScriptFromJson(
+	  {
+	    type: "all",
+	    scripts: [
+	      { type: "sig", keyHash: paymentCredential.hash },
+	      {
+		type: "before",
+		slot: lucid.utils.unixTimeToSlot(Date.now() + 1000000),
+	      },
+	    ],
+	  },
+	);
+
+	const policyId = lucid.utils.mintingPolicyToId(mintingPolicy);
+	const unit = policyId + fromText("nft");
+
+	const mintTx = await lucid.newTx()
+	     .mintAssets({ [unit]: 1n })
+	     .validTo(Date.now() + 200000)
+	     .attachMintingPolicy(mintingPolicy)
+	     .complete();
+
+	const signedMintTx = await mintTx.sign().complete();
+
+	const mintTxHash = await signedMintTx.submit();
+
+
+	const utxos = await lucid.utxosAt(
+	  recipient,
+	);
+	console.log(utxos);
```

Comment:
![image](https://user-images.githubusercontent.com/36756030/234787597-6c04a38c-c2a8-40d9-b472-18973224c874.png)

Reply:
Hello!

I'm bit stuck, maybe I'll get unstuck myself, but if you could review my pr😇

I added code for minting NFT to tests/lucid.test.ts L56;

It completes, but when I look at my wallet utxos, there isn't anything new there. L92

Using emulator.log(); didn't show anything interesting.

The best I could do is to print the minting transaction hash, L86

I suspect maybe I need to set the collateral in the wallet, right?

Thank you